Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Animations.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Animations
+    {
+        public int AnimationsId { get; set; }
+        public string Name { get; set; }
+        public string FileLocation { get; set; }
+        public string CoverImageLocation { get; set; }
+        public string Description { get; set; }
+        public ICollection<AnimationsWatched> AnimationWatched { get; set; }
+    }
+}
```

Comment:
Most of your model files need an empty line at the end of the file

Reply:
insert a line at 16? between last field and the } ?